setup.py

```diff
@@ -28,6 +28,7 @@
         packages=find_packages(exclude=("test*", "examples")),
         include_package_data=True,
         extras_require={"dev": development},
+        install_requires=["Flask>=0.10,<2", "marshmallow>=3.0,<4"],
```

Comment:
You've got two `install_requires` lines, guessing this occurred as a result of a bad merge.